ui/src/elements/ngm-auth.ts

```diff
@@ -64,29 +62,24 @@ export class NgmAuth extends LitElementI18n {
   }
 
   static readonly styles = css`
-  ngm-icon {
-    color: var(--color-bg);
-    background-color: var(--color-main);
-    width: 36px;
-    height: 36px;
-    border-radius: 50%;
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    cursor: pointer;
-  }
+    ngm-icon {
+      color: var(--color-bg);
+      background-color: var(--color-main);
+      width: 36px;
+      height: 36px;
+      border-radius: 50%;
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      cursor: pointer;
+    }
   `;
 
   render() {
     return html`
       <div
-           @click=${!this.user ? this.login : this.logout}>
-          <ngm-icon icon=${this.icon}></ngm-icon>
+           @click=${this.user ? this.logout : this.login}>
+          <ngm-icon icon='user'></ngm-icon>

```

Comment:
Use double quotes in HTML.
```suggestion
          <ngm-icon icon="user"></ngm-icon>

```